.github/workflows/build.yml

```diff
@@ -44,50 +44,11 @@ jobs:
             GIT_SHA=${{ github.sha }}
           platforms: linux/amd64
 
-  build-arm64:
-    runs-on: ubuntu-24.04-arm
-    permissions:
-      contents: write
-    steps:
-      - name: Checkout code
-        uses: actions/checkout@v4
-
-      - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v3
-
-      - name: Login to DockerHub
-        uses: docker/login-action@v3
-        with:
-          username: ${{ secrets.DOCKERHUB_USERNAME }}
-          password: ${{ secrets.DOCKERHUB_TOKEN }}
-
-      - name: Extract metadata for Docker
-        id: meta
-        uses: docker/metadata-action@v5
-        with:
-          images: audius/api
-          tags: |
-            type=raw,value=latest,enable=${{ github.ref == 'refs/heads/main' }},suffix=-arm64
-            type=sha,prefix=,suffix=-arm64
-
-      - name: Build and push
-        uses: docker/build-push-action@v5
-        with:
-          context: .
-          push: true
-          tags: ${{ steps.meta.outputs.tags }}
-          labels: ${{ steps.meta.outputs.labels }}
-          cache-from: type=registry,ref=audius/api:buildcache-arm64
-          cache-to: type=registry,ref=audius/api:buildcache-arm64,mode=max
-          build-args: |
-            GIT_SHA=${{ github.sha }}
-          platforms: linux/arm64
-        
   push-manifest:
     runs-on: ubuntu-latest
     permissions:
       contents: write
-    needs: [build-amd64, build-arm64]
```

Comment:
lets just do this and the below leave arm builds? when we see them green we can revert (otherwise hard to know we can turn back on)